remo/profiles/views.py

```diff
@@ -250,3 +257,21 @@ def delete_user(request, display_name):
         statsd.incr('profiles.delete_profile')
 
     return redirect('main')
+
+
+@cache_control(private=True)
+def list_alumni(request):
+    """List users in Alumni Group."""
+    query = User.objects.filter(groups__name='Alumni')
+
+    alumni_paginator = Paginator(query, settings.ITEMS_PER_PAGE)
+    alumni_page = request.GET.get('page', 1)
+
+    try:
+        alumni = alumni_paginator.page(alumni_page)
+    except PageNotAnInteger:
+        alumni = alumni_paginator.page(1)
+    except EmptyPage:
+        alumni = alumni_paginator.page(alumni_paginator.num_pages)
+
+    return render(request, 'profiles_list_alumni.html', {'alumni': alumni})
```

Comment:
Since you strip the macro you should pass the alumni object with the generic name `objects` since pagination.html expects it that way. And change the alumni template accordingly Isee above). Now the alumni page returns empty.